components/Dialogs/src/private/MDCAlertActionManager.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "MaterialButtons.h"
+#import "MaterialDialogs.h"
+
+@interface MDCAlertActionManager : NSObject
+
+@property(nonatomic, nonnull, strong, readonly) NSArray<MDCAlertAction *> *actions;
+/**
```

Comment:
Tip: Style: empty lines above and below all methods.

Reply:
Done